frontend/src/pages/Projects/Project/ProjectDetails/BudgetSection/FormConfig.ts

```diff
@@ -278,6 +399,46 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
   const updateUrl = `/projects/budget/${query?.data?.data?.data?.id}`;
   const deleteUrl = `/projects/budget/${query}`;
 
+  // validates that the total field does not exceed the detail_amount field
+  const validationSchema = object({
```

Comment:
did you mean to have yup validation into this ticket, not sure it was part of the scope....

Reply:
I'll answer your question with a question, did you look a the ticket requirements in JIRA?